py/client-ticking/build.gradle

```diff
@@ -174,26 +170,44 @@ def testPyClientTickingManyLinux = { wheelsSet, taskName, parentContainer, image
                   done
                   ''']
     copyOut {
-        into layout.buildDirectory.dir('test-results')
+        into layout.buildDirectory.dir(taskName + '-test-results')
     }
 }}
 
-def testPyClientTicking = testPyClientTickingManyLinux(
-    checksWheelSet,
-    'testPyClientTicking',
-    'pyClientTicking',
-    'ubi-minimal')
+def isCi = System.getenv().getOrDefault("CI", "false") == "true"
+
+def assembleWheelsSet = isCi
```

Comment:
Is this logic inverted?

It sounds the "true" case for "Is this CI" has more cases than the "false" case, where I would expect the opposite.

Or rather, what CI?  github check on every PR CI?  build artifacts for release CI?

Reply:
As written, this will cause any code that is executing in "CI" to have the full wheel set. If we want to limit it, to say "only build (/test) all of the wheels during the nightly build checks, or during release" there is some additional configuration logic we'll need to develop. That isn't something we've sliced-n-diced based on before.